### DISCUSSION TITLE
[UserStoriesToRefine] #5 As <user> I want a website where users can easily leave leave their feedback and suggestions

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#27] [General]
### As <user> I want a website where users can easily leave leave their feedback and suggestions

**Comment by christinagiancola** [comment]
I think there will be some variation here depending on what kind of feedback or suggestions we are hoping to collect unless we wanted users to submit feedback all at once in longer form or questionnaire. 

Acceptance Criteria:
1. When the user is choosing a language for the viewing the website and content, there is an option for the user to express that their preferred language is not listed as an option. 
2. At the end of watching a video from the video library, the user sees a pop-up/modal with a few questions regarding the quality and content of the video, including a question about the quality of the video translation if applicable. 
3. There is a 'feedback' link or button visible from all pages of the website that takes the user to a form where they can submit feedback, make suggestions, or ask questions about the website or content.
4. The user can quickly provide feedback about interest in video content by clicking one of two buttons near the video title, like a thumbs up/thumbs down (similar to youtube) or an up arrow/down arrow (similar to reddit).

**Comment by UnitedByChocolate** [comment]
As a [USER NAME] I can [ACHIEVE SOMETHING], so that [VALUE STATEMENT]
As a cacao producer I want a website where I can easily leave feedback and suggestions, so that I can impact future conference topics and presenters that are chosen to speak.

**Comment by christinagiancola** [comment]
As a [USER NAME] I can [ACHIEVE SOMETHING], so that [VALUE STATEMENT]

**Level 2: As a cacao producer I can input my primarily spoken language, so that I can consume content from the website.** 
Acceptance Criteria: When the user is choosing a language for the viewing the website and content, there is an option for the user to express that their preferred language is not listed as an option.


**Level 2: As a cacao producer I can provide feedback about the videos I watch, so that I am shown more relevant content as I continue to use the app.** 
Acceptance Criteria: At the end of watching a video from the video library, the user sees a pop-up/modal with a few questions regarding the quality and content of the video, including a question about the quality of the video translation if applicable.


**Level 2: As a cacao producer I can send a message directly to United by Chocolate, so that I can make suggestions, requests, or provide feedback easily.**
Acceptance Criteria: There is a 'feedback' link or button visible from all pages of the website that takes the user to a form where they can submit feedback, make suggestions, or ask questions about the website or content.

**Level 2: As a cacao producer I can quickly express my interest in video topics as I look through the library, so that I am shown content that is more relevant to my particular interests.** 
Acceptance Criteria: The user can quickly provide feedback about interest in video content by clicking one of two buttons near the video title, like a thumbs up/thumbs down (similar to youtube) or an up arrow/down arrow (similar to reddit).
